bin/printStream.py

```diff
@@ -44,17 +47,22 @@ def main():
     parser.add_argument('--group', type=str,
                         help='Globally unique name of the consumer group. '
                         'Consumers in the same group will share messages '
-                        '(i.e., only one consumer will receive a message, as in a queue).')
+                        '(i.e., only one consumer will receive a message, '
+                        'as in a queue).')
     parser.add_argument('--stampDir', type=str,
-                        help='Output directory for writing postage stamp cutout files.')
+                        help='Output directory for writing postage stamp'
+                        'cutout files.')
     avrogroup = parser.add_mutually_exclusive_group()
     avrogroup.add_argument('--decode', dest='avroFlag', action='store_true',
                            help='Decode from Avro format. (default)')
-    avrogroup.add_argument('--decode-off', dest='avroFlag', action='store_false',
+    avrogroup.add_argument('--decode-off', dest='avroFlag',
+                           action='store_false',
                            help='Do not decode from Avro format.')
     parser.set_defaults(avroFlag=True)
 
     args = parser.parse_args()
+    topic = args.topic
+    stampDir = args.stampDir
 
```

Comment:
I feel like "I did it to fit in our new column limit" is going to be the new code review excuse...